test/src/code_field/text_editing_value_test.dart

```diff
@@ -0,0 +1,381 @@
+          'No selection -> null',
+          value: TextEditingValue(text: loremIpsum),
+          expected: null,
+        ),
+
+        _StringExample(
+          'Word start',
+          value: TextEditingValue(
+            text: loremIpsum,
+            selection: TextSelection.collapsed(offset: 2),
+          ),
+          expected: 'Lorem',
+        ),
+
+        _StringExample(
+          '1 char',
+          value: TextEditingValue(
+            text: loremIpsum,
+            selection: TextSelection.collapsed(offset: 3),
+          ),
+          expected: 'Lorem',
+        ),
+
+        _StringExample(
+          '2 chars',
+          value: TextEditingValue(
+            text: loremIpsum,
+            selection: TextSelection.collapsed(offset: 4),
+          ),
+          expected: 'Lorem',
+        ),
+
+        _StringExample(
+          '3 chars',
+          value: TextEditingValue(
+            text: loremIpsum,
+            selection: TextSelection.collapsed(offset: 5),
+          ),
+          expected: 'Lorem',
+        ),
+
+        _StringExample(
+          '4 chars',
+          value: TextEditingValue(
+            text: loremIpsum,
+            selection: TextSelection.collapsed(offset: 6),
+          ),
+          expected: 'Lorem',
+        ),
+
+        _StringExample(
+          'Just after a word',
+          value: TextEditingValue(
+            text: loremIpsum,
+            selection: TextSelection.collapsed(offset: 7),
+          ),
+          expected: 'Lorem',
+        ),
+
+        _StringExample(
+          'Non-collapsed -> null',
+          value: TextEditingValue(
+            text: loremIpsum,
+            selection: TextSelection(baseOffset: 3, extentOffset: 4),
+          ),
+          expected: null,
+        ),
+
+        _StringExample(
+          'String start',
+          value: TextEditingValue(
+            text: loremIpsum,
+            selection: TextSelection.collapsed(offset: 0),
+          ),
+          expected: '1',
+        ),
+
+        _StringExample(
+          'Non-word string start',
+          value: TextEditingValue(
+            text: '   ',
+            selection: TextSelection.collapsed(offset: 0),
+          ),
+          expected: null,
+        ),
+
+        _StringExample(
+          'At the last word',
+          value: TextEditingValue(
+            text: 'abc def',
+            selection: TextSelection.collapsed(offset: 6),
+          ),
+          expected: 'def',
+        ),
+
+        _StringExample(
+          'After the last word',
+          value: TextEditingValue(
+            text: 'abc def',
+            selection: TextSelection.collapsed(offset: 7),
+          ),
+          expected: 'def',
+        ),
+      ];
+
+      for (final example in examples) {
+        final result = example.value.wordAtCursor;
+
+        expect(result, example.expected, reason: example.name);
+      }
+    });
+
+    test('wordToCursor', () {
+      const examples = [
+        //
```

Comment:
Are these empty comments a part of examples? If so, is it clear for others?

Reply:
This forces the Dart formatter to preserve spaces between examples.
It is not clear for others.